README.md

```diff
@@ -81,69 +81,116 @@ Summary of one-time project setup:
 
 ## Project Development Workflow
 
+We will use a Test Driven Development programming workflow to work on this project. Notice the Red-Green-Refactor steps in the workflow steps outlined below.
+
 1. When you want to begin work on this project, ensure that your virtual environment is activated:
 
 ```bash
 $ source venv/bin/activate
 ```
 
-2. Find the test file that contains the test you want to run.
+2. Check the `tests/unit_tests` folder, and find the test file you want to run
 
-   - Check the `tests` folder, and find the test file you want to run
    - In that test file, read through each test case
+      - If it is incomplete, complete the test. **There are incomplete tests for Waves 01, 02, and 06**. 
+        - *Is this a nominal or edge case?*
+        - *What type of input do we need to test this case?*
+        - *What is the expected output for the given input?*
+   - Remove the lines that contain `@pytest.mark.skip()` for the test(s) you want to run.
 
-3. Run the tests for your specific wave
+3. Run the test(s)! (RED)
+    - *See the [Details About How to Run Tests](#details-about-how-to-run-tests) section below for more information on how to run test(s).*
 
 ```bash
-# Must be in activated virtual environment
-$ pytest tests/test_wave_01.py
+# Must be in activated virtual environment in the project-root directory
+$ pytest
 ```
 
-4. Focus on the top test failure. Read through the test failure, and understand why the failure is happening. Confirm your findings with a classmate.
+4. Read through the test failure, and understand why the failure is happening. Confirm your findings with a classmate. 
+    - If it is a test you wrote, consider whether you are actually testing what you intend to test. Does the test need modification?
+
+5. Make a plan to implement code to pass the test.
 
-5. Make a plan to fix the test failure.
+6. Write code to pass the test (GREEN).
 
-6. Write code to fix the test failure.
+7. Re-run the test(s).
 
-7. Re-run the tests.
+8. Repeat steps 3-7 until that test passes!
 
-8. Repeat steps 5-7 until that test passes!
+9. Consider looking for opportunities to improve your code (REFACTOR)
 
-9. Repeats steps 4-8 until you have finished all tests in the file.
+10. Repeats steps 2-8 until you have finished all tests in the file.
```

Comment:
Couple small things:
- Should it be steps 2-9 to include the refactor step?
- I think "Repeats" should be "Repeat"